src/api/operations/project.queries.graphql

```diff
@@ -123,6 +123,34 @@ query GetProjectServices(
   }
 }
 
+query GetProjectAssessments(
+  $id: ID!
+  $limit: Int = 10
+  $offset: Int = 0
+  $sortOrder: AssessmentSortOption = ASSESSMENT_DATE
+  $filters: AssessmentFilterOptions = null
+) {
+  project(id: $id) {
+    id
+    assessments(
+      limit: $limit
+      offset: $offset
+      sortOrder: $sortOrder
+      filters: $filters
```

Comment:
Wanted to note that filtering by 'Custom Assessment' type didn't work how I expected. It returned no results despite the presence of some custom assessments. @gigxz is this a bug or due to my misunderstanding?

This issue seemed to impact the existing assessments tables (client, enrollment) as well as the newly built one, so, if it's a bug I'll create a new PT bugfix ticket.

Reply:
It's a big - pt story https://www.pivotaltracker.com/story/show/187400755